### ISSUE TITLE
Replace noNodes with numNodes?

### ISSUE THREAD

**Issue by tomjaguarpaw** [haskell/fgl#58]
Is there any chance that `numNodes` could be added as a synonym for `noNodes` (and perhaps `noNodes` deprecated)?

`noNodes` really looks like "no nodes" and intuitively seems to suggest a `Bool` valued function which tells you whether there are no nodes in the graph!

**Comment by ivan-m** [comment]
Yeah, I've been bitten by this before. My preference however is yo use `order` instead.

**Comment by tomjaguarpaw** [comment]
`order` is better than `noNodes` though perhaps less familiar to people who haven't studied graph theory.

**Comment by ntc2** [comment]
I prefer `numNodes` to `order`, but either is fine, and both are much better than `noNodes` (I had the same `noNodes` means `empty` confusion the first time I scanned the docs).
